cli/tests/tobs-cli-tests/portforward_test.go

```diff
@@ -27,10 +27,19 @@ func testpf(t testing.TB, timescale, grafana, prometheus, promscale, promlens st
 		cmds = append(cmds, "-l", promlens)
 	}
 
+	// kubectl get pvc -A
+	out := exec.Command("kubectl", "get", "pods", "-A")
+	output, err := out.CombinedOutput()
+	if err != nil {
+		t.Log(string(output))
+		t.Fatal(err)
+	}
+	t.Log(string(output))
```

Comment:
This line is done if there is an error or not, can be done before error check if that is the case.